diff --git a/importmap.php b/importmap.php
index 9be0d4c..0c489cc 100644
--- a/importmap.php
+++ b/importmap.php
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * Returns the import map for this application.
+ * Returns the importmap for this application.
  *
  * - "path" is a path inside the asset mapper system. Use the
  *     "debug:asset-map" command to see the full list of paths.
@@ -18,4 +18,7 @@ return [
         'path' => './assets/app.js',
         'entrypoint' => true,
     ],
+    'js-confetti' => [
+        'version' => '0.11.0',
+    ],
 ];
