diff --git a/assets/controllers/modal_controller.js b/assets/controllers/modal_controller.js
index 9ab1317..280728c 100644
--- a/assets/controllers/modal_controller.js
+++ b/assets/controllers/modal_controller.js
@@ -1,7 +1,7 @@
 import { Controller } from '@hotwired/stimulus';
 
 export default class extends Controller {
-    static targets = ['dialog', 'content'];
+    static targets = ['dialog', 'content', 'loadingTemplate'];
 
     observer = null;
 
@@ -50,4 +50,13 @@ export default class extends Controller {
             this.dialogTarget.close();
         }
     }
+
+    showLoading(event) {
+        // do nothing if the dialog is already open
+        if (this.dialogTarget.open) {
+            return;
+        }
+
+        this.contentTarget.innerHTML = this.loadingTemplateTarget.innerHTML;
+    }
 }
diff --git a/src/Controller/VoyageController.php b/src/Controller/VoyageController.php
index 3adad4b..ceb0b8c 100644
--- a/src/Controller/VoyageController.php
+++ b/src/Controller/VoyageController.php
@@ -26,6 +26,7 @@ class VoyageController extends AbstractController
     #[Route('/new', name: 'app_voyage_new', methods: ['GET', 'POST'])]
     public function new(Request $request, EntityManagerInterface $entityManager): Response
     {
+        sleep(1);
         $voyage = new Voyage();
         $form = $this->createForm(VoyageType::class, $voyage, [
             'action' => $this->generateUrl('app_voyage_new'),
diff --git a/templates/base.html.twig b/templates/base.html.twig
index 5bd20fa..299316a 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -58,6 +58,7 @@
             <turbo-frame
                 id="modal"
                 data-modal-target="content"
+                data-action="turbo:before-fetch-request->modal#showLoading"
             ></turbo-frame>
             <twig:block name="loading_template">
                 <div class="bg-space-pattern bg-cover rounded-lg p-8">
