diff --git a/templates/voyage/new.html.twig b/templates/voyage/new.html.twig
index 4970a6c..2eb86fd 100644
--- a/templates/voyage/new.html.twig
+++ b/templates/voyage/new.html.twig
@@ -24,8 +24,6 @@
 
 {% block success_stream %}
     <turbo-stream action="update" target="modal">
-        <template>
-            Please close me!
-        </template>
+        <template></template>
     </turbo-stream>
 {% endblock %}
