diff --git a/tailwind.config.js b/tailwind.config.js
index ce266a5..46f13e7 100644
--- a/tailwind.config.js
+++ b/tailwind.config.js
@@ -7,7 +7,17 @@ module.exports = {
     "./src/Twig/Components/**/*.php"
   ],
   theme: {
-    extend: {},
+    extend: {
+      animation: {
+        'fade-in': 'fadeIn .5s ease-out;',
+      },
+      keyframes: {
+        fadeIn: {
+          '0%': { opacity: 0 },
+          '100%': { opacity: 1 },
+        },
+      },
+    },
   },
   plugins: [],
 }
diff --git a/templates/voyage/_row.html.twig b/templates/voyage/_row.html.twig
index 1ba68b3..e728ee9 100644
--- a/templates/voyage/_row.html.twig
+++ b/templates/voyage/_row.html.twig
@@ -1,4 +1,4 @@
-<tr class="even:bg-gray-700 odd:bg-gray-600">
+<tr class="even:bg-gray-700 odd:bg-gray-600 {{ animateIn|default ? 'animate-fade-in' : '' }}">
     <td class="px-6 py-4 whitespace-nowrap">{{ voyage.id }}</td>
     <td class="px-6 py-4">{{ voyage.purpose }}</td>
     <td class="px-6 py-4 whitespace-nowrap">{{ voyage.leaveAt ? voyage.leaveAt|date('Y-m-d H:i:s') : '' }}</td>
diff --git a/templates/voyage/new.html.twig b/templates/voyage/new.html.twig
index 63c2edc..634f738 100644
--- a/templates/voyage/new.html.twig
+++ b/templates/voyage/new.html.twig
@@ -32,6 +32,8 @@
     </turbo-stream>
 
     <turbo-stream action="prepend" targets="#voyage-crud-table tbody">
-        <template>{{ include('voyage/_row.html.twig') }}</template>
+        <template>{{ include('voyage/_row.html.twig', {
+            animateIn: true
+        }) }}</template>
     </turbo-stream>
 {% endblock %}
