diff --git a/templates/main/homepage.html.twig b/templates/main/homepage.html.twig
index 5877d7c..7640f19 100644
--- a/templates/main/homepage.html.twig
+++ b/templates/main/homepage.html.twig
@@ -17,6 +17,7 @@
                 action="{{ path('app_homepage') }}"
                 class="mb-6 flex justify-between"
                 data-turbo-frame="voyage-list"
+                data-controller="autosubmit"
             >
                 <div class="w-1/3 flex">
                     <input
@@ -27,6 +28,7 @@
                         placeholder="Search voyages"
                         class="px-4 py-2 rounded bg-gray-800 text-white placeholder-gray-400"
                         autofocus
+                        data-action="autosubmit#debouncedSubmit"
                     >
                 </div>
 
