diff --git a/templates/main/homepage.html.twig b/templates/main/homepage.html.twig
index e0cab57..d3bae81 100644
--- a/templates/main/homepage.html.twig
+++ b/templates/main/homepage.html.twig
@@ -31,7 +31,7 @@
 
                 <div class="whitespace-nowrap m-2 mr-4">{{ voyages|length }} results</div>
             </form>
-            <div>
+            <turbo-frame id="voyage-list">
                 <div class="bg-gray-800 p-4 rounded">
                     <table class="w-full text-white">
                         <thead>
@@ -85,7 +85,7 @@
                         Page {{ voyages.currentPage }}/{{ voyages.nbPages }}
                     </div>
                 </div>
-            </div>
+            </turbo-frame>
         </section>
     </div>
 {% endblock %}
