diff --git a/templates/main/homepage.html.twig b/templates/main/homepage.html.twig
index 5877d7c..2a3f0c4 100644
--- a/templates/main/homepage.html.twig
+++ b/templates/main/homepage.html.twig
@@ -12,6 +12,14 @@
         </aside>
 
         <section class="flex-1 ml-10">
+            <dialog open>
+                I am a dialog!
+
+                <form method="dialog">
+                    <button>Close</button>
+                </form>
+            </dialog>
+
             <form
                 method="GET"
                 action="{{ path('app_homepage') }}"
