diff --git a/templates/main/homepage.html.twig b/templates/main/homepage.html.twig
index 7434753..52e1711 100644
--- a/templates/main/homepage.html.twig
+++ b/templates/main/homepage.html.twig
@@ -30,7 +30,17 @@
                         autofocus
                         data-action="autosubmit#debouncedSubmit"
                     >
-                    {{ include('main/_search_filters.html.twig') }}
+
+                    <twig:Modal>
+                        <twig:block name="trigger">
+                            <twig:Button
+                                variant="link"
+                                type="button"
+                                data-action="modal#open"
+                            >Options</twig:Button>
+                        </twig:block>
+                        {{ include('main/_search_filters.html.twig') }}
+                    </twig:Modal>
                 </div>
 
                 <div class="whitespace-nowrap m-2 mr-4">{{ voyages|length }} results</div>
