diff --git a/templates/main/homepage.html.twig b/templates/main/homepage.html.twig
index 52e1711..743be1c 100644
--- a/templates/main/homepage.html.twig
+++ b/templates/main/homepage.html.twig
@@ -40,6 +40,13 @@
                             >Options</twig:Button>
                         </twig:block>
                         {{ include('main/_search_filters.html.twig') }}
+
+                        <div class="flex justify-end">
+                            <twig:Button
+                                variant="success"
+                                data-action="modal#close"
+                            >See Results</twig:Button>
+                        </div>
                     </twig:Modal>
                 </div>
 
