diff --git a/assets/controllers/modal_controller.js b/assets/controllers/modal_controller.js
new file mode 100644
index 0000000..084b4ea
--- /dev/null
+++ b/assets/controllers/modal_controller.js
@@ -0,0 +1,21 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+    static targets = ['dialog'];
+
+    open() {
+        this.dialogTarget.showModal();
+        document.body.classList.add('overflow-hidden', 'blur-sm');
+    }
+
+    close() {
+        this.dialogTarget.close();
+        document.body.classList.remove('overflow-hidden', 'blur-sm');
+    }
+
+    clickOutside(event) {
+        if (event.target === this.dialogTarget) {
+            this.dialogTarget.close();
+        }
+    }
+}
