diff --git a/assets/lib/greeting.js b/assets/lib/greeting.js
new file mode 100644
index 0000000..598dbe7
--- /dev/null
+++ b/assets/lib/greeting.js
@@ -0,0 +1,3 @@
+export default function greeting(Ryan) {
+    console.log(`Hello ${Ryan}!`);
+}
diff --git a/templates/base.html.twig b/templates/base.html.twig
index 56a9fe3..b4a68fd 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -10,6 +10,11 @@
 
         {% block javascripts %}
             {{ importmap('app') }}
+            <script type="module">
+                import greet from '{{ asset('lib/greeting.js') }}';
+
+                greet('Ryan');
+            </script>
         {% endblock %}
     </head>
     <body>
