diff --git a/templates/base.html.twig b/templates/base.html.twig
index acd540b..b1be1ed 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -20,6 +20,8 @@
                         <!-- Logo -->
                         <a
                             href="{{ path('app_homepage') }}"
+                            data-controller="celebrate"
+                            data-action="mouseover->celebrate#poof"
                         >
                             <img src="{{ asset('images/logo.png') }}" width="50" alt="Space Inviters Logo" >
                         </a>
