diff --git a/templates/voyage/_form.html.twig b/templates/voyage/_form.html.twig
index 892208b..5123334 100644
--- a/templates/voyage/_form.html.twig
+++ b/templates/voyage/_form.html.twig
@@ -1,5 +1,5 @@
 {{ form_start(form) }}
     {{ form_widget(form) }}
 
-    <twig:Button formnovalidate>{{ button_label|default('Save') }}</twig:Button>
+    <twig:Button variant="success" formnovalidate>{{ button_label|default('Save') }}</twig:Button>
 {{ form_end(form) }}
